pkg/api/meta/metatypes/deep_copy_generated.go

```diff
@@ -1,33 +0,0 @@
-// +build !ignore_autogenerated
```

Comment:
It does not appear to be used in any path that does deep copies
(instead, it's copied by the garbage collector when it calls
meta.Accessor).  I think it's safe to remove.

I'm going to in a follow on print debug info if we copy something
without generated structs.  However, the reflect copy path should be
fast for this type.